adapt-and-extend/theming/resources.rst

```diff
@@ -4,10 +4,11 @@ JS/CSS Resources
 
 .. admonition:: Description
 
-    Resources are js/css and its dependencies to add the frontend logic and design
-    on top of Plone. There are so many different kind of resources like widgets, design styles,
-    behavior logic and single page apps. In order to organize them we are using two
-    main standard technologies: require.js and less.
+    Resources are JavaScript, CSS or LESS resources and their dependencies.
```

Comment:
AFAIK, in the resource registry only JavaScript, CSS and LESS are managed at the moment. Plain files can still be referenced in CSS/LESS/JS via static or resource directories.